pymongo/collection.py

```diff
@@ -2495,6 +2495,7 @@ def watch(
         start_after: Optional[Mapping[str, Any]] = None,
         comment: Optional[Any] = None,
         full_document_before_change: Optional[str] = None,
+        show_expanded_events: Optional[bool] = None,
```

Comment:
Needs docs and versionadded.  Looks like we missed `full_document_before_change` in the docstring earlier as well.